data_dictionary/tables.md

```diff
@@ -159,20 +150,20 @@ This a single visit to the hospital.
 
 ### **Attributes/Column Headers**
 
-| Name | Type | Description |
-|---| --- |---|
-| hospitalVisitId | bigint | Unique identifier in EMAP for this **HospitalVisit** record. |
-| mrnId | [Mrn](#Mrn) | Identifier for the [Mrn](#Mrn) associated with this record. |
-| sourceSystem | varchar(255) | The source system from which we learnt about this hospital visit. |
-| presentationTime | timestamp without timezone | Date and time at which this **HospitalVisit** was first recorded. |
-| admissionTime | timestamp without timezone | Date and time at which this **HospitalVisit** formally began. |
-| dischargeTime | timestamp without timezone | Date and time at which this **HospitalVisit** formally ended. |
-| patientClass | varchar(255) | The patient class. E.g. Inpatient or Outpaitent. |
-| arrivalMethod | varchar(255) | The patient's arrival method at hospital. |
+| Name                 | Type | Description |
+|----------------------| --- |---|
+| hospitalVisitId      | bigint | Unique identifier in EMAP for this **HospitalVisit** record. |
+| mrnId                | [Mrn](#Mrn) | Identifier for the [Mrn](#Mrn) associated with this record. |
+| sourceSystem         | varchar(255) | The source system from which we learnt about this hospital visit. |
+| presentationDatetime | timestamp without timezone | Date and time at which this **HospitalVisit** was first recorded. |
+| admissionDaetime     | timestamp without timezone | Date and time at which this **HospitalVisit** formally began. |

```

Comment:
```suggestion
| admissionDatetime     | timestamp without timezone | Date and time at which this **HospitalVisit** formally began. |

```

Reply:
wait, is this not all automatically generated? 

Also, `timestamp without timezone?` All of these should have timezones